core/async_job/rafted_value_config_maker.ex

```diff
@@ -6,17 +6,17 @@ defmodule AntikytheraCore.AsyncJob.RaftedValueConfigMaker do
   @behaviour RaftFleet.RaftedValueConfigMaker
 
   @options [
-    heartbeat_timeout:                   1_000,
-    election_timeout:                    5_000,
-    election_timeout_clock_drift_margin:   500,
+    heartbeat_timeout: 1_000,
+    election_timeout: 5_000,
+    election_timeout_clock_drift_margin: 500
   ]
-  defun options() :: Keyword.t, do: @options
+  defun(options() :: Keyword.t(), do: @options)
```

Comment:
I hate...:-1:

Reply:
Is this problem because we use `defun` instead of `fun`?

Reply:
Yes, it looks like it can be worked around by using `def` or using a `do/end` block.

Reply:
Maybe so.
The following is the result of `mix format`.
* from
```
use Croma

defmodule Foo do
  @options [1]
  defun options() :: Keyword.t, do: @options
end

defmodule Bar do
  @options [1]
  def options() :: Keyword.t, do: @options
end
```
* diff
```diff
 defmodule Foo do
   @options [1]
-  defun options() :: Keyword.t, do: @options
+  defun(options() :: Keyword.t(), do: @options)
 end

 defmodule Bar do
   @options [1]
-  def options() :: Keyword.t, do: @options
+  def options() :: Keyword.t(), do: @options
 end
```

Reply:
Does `:locals_without_parens` option solve this?

Reply:
`:locals_without_parens` works well!
`diff` is the following.
```diff
 defmodule Foo do
   @options [heartbeat_timeout: 1_000]
-  defun options() :: Keyword.t, do: @options
+  defun options() :: Keyword.t(), do: @options
 end

 defmodule Bar do
   @options [heartbeat_timeout: 1_000]
-  def options() :: Keyword.t, do: @options
+  def options() :: Keyword.t(), do: @options
 end
```